src/pyk/proof/equality.py

```diff
@@ -173,6 +162,89 @@ def summary(self) -> Iterable[str]:
         ]
 
 
+class RefutationProof(Proof):
+    def __init__(
+        self,
+        id: str,
+        sort: KSort,
+        constraints: Iterable[KInner] = (),
+        csubst: CSubst | None = None,
+        simplified_constraints: KInner | None = None,
+        proof_dir: Path | None = None,
+    ):
+        super().__init__(id, proof_dir=proof_dir)
+        self.sort = sort
+        self.constraints = tuple(constraints)
+        self.csubst = csubst
+        self.simplified_constraints = simplified_constraints
+
+    def add_constraint(self, new_constraint: KInner) -> None:
+        self.constraints = (*self.constraints, new_constraint)
+
+    def set_simplified_constraints(self, simplified: KInner) -> None:
+        self.simplified_constraints = simplified
+
+    def set_csubst(self, csubst: CSubst) -> None:
+        self.csubst = csubst
+
+    @property
+    def status(self) -> ProofStatus:
+        if self.simplified_constraints is None:
+            return ProofStatus.PENDING
+        elif self.csubst is not None:
+            return ProofStatus.FAILED
+        else:
+            return ProofStatus.PASSED
+
+    @property
+    def dict(self) -> dict[str, Any]:
+        dct = {
+            'type': 'RefutationProof',
+            'id': self.id,
+            'sort': self.sort.to_dict(),
+            'constraints': [c.to_dict() for c in self.constraints],
+        }
+        if self.simplified_constraints is not None:
+            dct['simplified_constraints'] = self.simplified_constraints.to_dict()
+        if self.csubst is not None:
+            dct['csubst'] = self.csubst.to_dict()
+        return dct
+
+    @classmethod
+    def from_dict(cls: type[RefutationProof], dct: Mapping[str, Any], proof_dir: Path | None = None) -> RefutationProof:
+        id = dct['id']
+        sort = KSort.from_dict(dct['sort'])
+        constraints = [KInner.from_dict(c) for c in dct['constraints']]
+        simplified_constraints = (
+            KInner.from_dict(dct['simplified_constraints']) if 'simplified_constraints' in dct else None
+        )
+        csubst = CSubst.from_dict(dct['csubst']) if 'csubst' in dct else None
+        return RefutationProof(
+            id=id,
+            sort=sort,
+            constraints=constraints,
+            csubst=csubst,
+            simplified_constraints=simplified_constraints,
+            proof_dir=proof_dir,
+        )
+
+    @property
+    def summary(self) -> Iterable[str]:
+        return [
+            f'RefutationProof: {self.id}',
+            f'  status: {self.status}',
+        ]
```

Comment:
```suggestion
    @property
    def summary(self) -> Iterable[str]:
        return [
            f'RefutationProof: {self.id}',
            f'    status: {self.status}',
        ]
```

Are we using 2 spaces or 4 on other proof summaries?

Reply:
It's now consistently 4 spaces for all `summary`s.